samples/SampleApp/Startup.cs

```diff
@@ -46,6 +50,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory, IAp
                     context.Request.Path,
                     context.Request.QueryString);
 
+                foreach (var q in context.Request.Query) { }
```

Comment:
Was to force that feature interface to be created, but doesn't make sense to keep as part of the sample. Removed.